openfe/tests/setup/test_molecule.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+
+from openfe.setup import Molecule
+from rdkit import Chem
+
+
+@pytest.fixture
+def alt_ethane():
+    return Molecule(Chem.MolFromSmiles("CC"))
+
+
+@pytest.fixture
+def named_ethane():
+    mol = Chem.MolFromSmiles("CC")
+    mol.SetProp("_Name", "ethane")
+    return Molecule(mol)
+
+
+class TestMolecule:
+    def test_rdkit_behavior(self, ethane, alt_ethane):
+        # Check that fixture setup is correct (we aren't accidentally
+        # testing tautologies) and that rdkit continues to behave in a way
+        # that require our custom equality.
+        assert ethane is not alt_ethane
+        assert ethane.rdkit is not alt_ethane.rdkit
+        assert ethane.rdkit != alt_ethane.rdkit  # rdkit might change this
```

Comment:
we don't have to test this, it's not our behaviour

Reply:
removed the last check, which is rdkit dependent. Other two are to ensure that the tests don't give false passes if something changes in fixtures.